context/app/static/js/components/NoticeAlert.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import Alert from '@material-ui/lab/Alert';
+import AlertTitle from '@material-ui/lab/AlertTitle';
+import Container from '@material-ui/core/Container';
+
+export default function NoticeAlert(props) {
+  const generateErrorList = () => props.errors.map((errorObj, ind) => {
+    if (errorObj.issue_url.length) {
+      const base = 'https://github.com/hubmapconsortium/';
+      const title = encodeURIComponent('Validation failure');
+      let body = encodeURIComponent(`We have this validation error: ${errorObj.traceback}`);
+      if (body.length > 1500) {
+        body = `Note, traceback has been trimmed to 1500 chars./n ${body.slice(1500)}`;
+      }
+      // Body trim is a quick fix to avoid 414 error (2000 char limit).
```

Comment:
@mccalluc The preferred solution would be a post and redirect, which is why trimming here would be more appropriate. But, I can trim at the source since it is a quick fix for now.